tests/testdata/testdata.go

```diff
@@ -33,6 +33,8 @@ const (
 	ClusterDisplayName2 = "Cluster 2"
 	ClusterDisplayName3 = "Cluster 3"
 
+	OrgID = testdata.OrgID
```

Comment:
i don't see this being used anywhere

Reply:
I added it for the upgrade_risks_prediction tests in order to avoid using both testdata from smart proxy and from aggregator data just for 1 variable